### PR TITLE
ADEN-6313 Add i18n to UAP HiVi

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: src/common/i18n.js [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+		},
+		'zh-hant': {
+			advertisement: '廣告',
+			'learn-more': '閱讀更多'
+		}
+	}
+};
+
+export function getTranslation(category, key) {
+	const language = window.wgContentLanguage && typeof i18n[category][window.wgContentLanguage] !== 'undefined'
```
Comment (fraszczakszymon) [inline src/common/i18n.js]: Don't use window to store that please. What about context-service?

File: src/common/i18n.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+		? window.wgContentLanguage : defaultLanguage;
+
+	return i18n[category][language][key] || i18n[category][defaultLanguage][key];
```
Comment (fraszczakszymon) [inline src/common/i18n.js]: OR seems to be not needed. `language` is going to be `defaultLanguage` in L160, right?

File: src/common/i18n.js (hunk 1 above)
Comment (fraszczakszymon) [reply src/common/i18n.js]: `context.get('options.contentLanguage')`

File: src/common/i18n.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
+const defaultLanguage = 'en';
+
+const i18n = {
+	labels: {
+		en: {
+			advertisement: 'Advertisement',
+			'learn-more': 'Learn More'
```
Comment (fraszczakszymon) [inline src/common/i18n.js]: Should we request more translations for 'Learn more'? What about some follow up ticket

File: src/common/i18n.js [hunk 4]
```diff
@@ -0,0 +1,164 @@
+const defaultLanguage = 'en';
+
+const i18n = {
```
Comment (fraszczakszymon) [inline src/common/i18n.js]: Could you move this const to separate file please? Look for other `constants.js` files

File: src/common/i18n.js (hunk 1 above)
Comment (dmnsobczak) [reply src/common/i18n.js]: Ok, done.

File: src/common/i18n.js (hunk 3 above)
Comment (dmnsobczak) [reply src/common/i18n.js]: Good idea, I will create a ticket.

File: src/common/i18n.js (hunk 4 above)
Comment (dmnsobczak) [reply src/common/i18n.js]: Done.

**Review by fraszczakszymon — APPROVED** [review]

**Review by leshniak — APPROVED** [review]

**Pull request merged**